src/App.tsx

```diff
@@ -8,7 +8,52 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './services/todos';
+
+import { Todo } from './types/Todo';
+import { Visible } from './types/Visible';
+
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [isListLoading, setIsListLoading] = useState<boolean>(false);
+  const [listErrorMessage, setListErrorMessage] = useState<string>('');
+
+  const [visible, setVisible] = useState<Visible>('all');
+  const [filterBy, setFilterBy] = useState<string>('');
+
+  const [activeTodo, setActiveTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    setIsListLoading(true);
+
+    getTodos()
+      .then(data => {
+        setTodoList(data);
+      })
+      .catch(() => setListErrorMessage('Try again Later'))
```

Comment:
The error message in the `catch` block is generic. Consider logging the error or providing a more specific message to help with debugging.